model/pipeline/src/main/proto/schema.proto

```diff
@@ -31,6 +31,7 @@ option java_outer_classname = "SchemaApi";
 message Schema {
   repeated Field fields = 1;
   string id = 2;
+  map<string, Option> options = 3;
```

Comment:
This fields redundant since Option already contains a name. Maybe just make repeated?

Reply:
Looking back to the history I see that it was a repeated option. I don't have a preference. I can set it back to repeated option.